README.md

```diff
@@ -38,11 +38,28 @@ You should then be able to run `bundle install` as usual.
 
 At the moment Travis CI is available as two separate sites - https://travis-ci.org for Open Source
 projects and https://travis-ci.com for private projects. travis-web will connect
-to the Open Source version by default. In order to connect it to the API for private projects
-you need to run:
+to the Open Source version by default. In order to connect it to the API for private projects you need to setup your `.env` file.
+
+Copy `.env.example` to `.env`
 
 ```
-TRAVIS_PRO=true ember serve --ssl --ssl-key=ssl/server.key --ssl-cert=ssl/server.crt
+$ cp .env.example .env
+```
+
+Set the required `env` variables to run in private mode:
+
+```
+API_ENDPOINT=https://api-staging.travis-ci.com
```

Comment:
If we comment out the `API_ENDPOINT` by default in `.env`, then no need to change it here.

Reply:
Also I think we can remove `STRIPE_PUBLISHABLE_KEY` and `TRAVIS_ENTERPRISE` - as they don't affect private repos and thus don't need to be changed.

Reply:
Yes, that makes sense. Updated. 👍